frontend/src/v5/ui/components/shared/userMenu/editProfileModal/editProfilePersonalTab/editProfilePersonalTab.component.tsx

```diff
@@ -122,75 +121,73 @@ export const EditProfilePersonalTab = ({
 
 	return (
 		<ScrollArea>
-			<ScrollAreaPadding>
-				<EditProfileAvatar user={user} />
-				<FormTextField
-					name="firstName"
-					control={control}
-					label={formatMessage({
-						id: 'editProfile.form.firstName',
-						defaultMessage: 'First Name',
-					})}
-					required
-					formError={formErrors.firstName}
-				/>
-				<FormTextField
-					name="lastName"
-					control={control}
-					label={formatMessage({
-						id: 'editProfile.form.lastName',
-						defaultMessage: 'Last Name',
-					})}
-					required
-					formError={formErrors.lastName}
-				/>
-				<FormTextField
-					name="email"
-					control={control}
-					label={formatMessage({
-						id: 'editProfile.form.email',
-						defaultMessage: 'Email',
-					})}
-					required
-					formError={formErrors.email}
-				/>
-				<FormTextField
-					name="company"
-					control={control}
-					label={formatMessage({
-						id: 'editProfile.form.company',
-						defaultMessage: 'Company',
-					})}
-					formError={formErrors.company}
-				/>
-				<FormSelect
-					name="countryCode"
-					control={control}
-					label={formatMessage({
-						id: 'userSignup.form.countryCode',
-						defaultMessage: 'Country',
-					})}
-					required
-				>
-					{clientConfigService.countries.map((country) => (
-						<MenuItem key={country.code} value={country.code}>
-							{country.name}
-						</MenuItem>
-					))}
-				</FormSelect>
-				{submitWasSuccessful && (
-					<SuccessMessage>
-						<FormattedMessage
-							id="editProfile.form.success"
-							defaultMessage="Your profile has been changed successfully."
-						/>
-					</SuccessMessage>
-				)}
-				<UnhandledError
-					error={unexpectedError}
-					expectedErrorValidators={[emailAlreadyExists, isFileFormatUnsupported]}
-				/>
-			</ScrollAreaPadding>
+			<EditProfileAvatar user={user} />
+			<FormTextField
+				name="firstName"
+				control={control}
+				label={formatMessage({
+					id: 'editProfile.form.firstName',
+					defaultMessage: 'First Name',
+				})}
+				required
+				formError={formErrors.firstName}
+			/>
+			<FormTextField
+				name="lastName"
+				control={control}
+				label={formatMessage({
+					id: 'editProfile.form.lastName',
+					defaultMessage: 'Last Name',
+				})}
+				required
+				formError={formErrors.lastName}
+			/>
+			<FormTextField
+				name="email"
+				control={control}
+				label={formatMessage({
+					id: 'editProfile.form.email',
+					defaultMessage: 'Email',
+				})}
+				required
+				formError={formErrors.email}
+			/>
+			<FormTextField
+				name="company"
+				control={control}
+				label={formatMessage({
+					id: 'editProfile.form.company',
+					defaultMessage: 'Company',
+				})}
+				formError={formErrors.company}
+			/>
+			<FormSelect
+				name="countryCode"
+				control={control}
+				label={formatMessage({
+					id: 'userSignup.form.countryCode',
```

Comment:
I know this isn't something you changed, but shouldn't this id be `editProfile.form.countryCode`?

Reply:
Fair point, I'll fix it!

Reply:
Done, I have renamed this one as well as the other IDs in the edit profile modal to share a common format